test/run-e2e-ocp.sh

```diff
@@ -64,6 +64,7 @@ delete_obo() {
 
 	oc delete -f hack/olm/subscription.yaml || true
 	oc delete -f hack/olm/catalog-src.yaml || true
+	oc delete crds $(oc api-resources --api-group=monitoring.rhobs -o name)
```

Comment:
As Simon comments, please change this to let the linter go green, thanks!

```suggestion
	oc delete crds "$(oc api-resources --api-group=monitoring.rhobs -o name)"
```